apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/Emulator/metadata_server.py

```diff
@@ -0,0 +1,353 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import OrderedDict
+from flask import Flask, request
+from werkzeug.wrappers import Request
+import common, errno, getopt, json, logging, os, socket, sys, time
+
+numeric_project_id = 12345;
+numeric_project_id_token = "<NUMERIC_PROJECT_ID>";
+project_id = 'fake-project';
+project_id_token = "<PROJECT_ID>";
+
+app = Flask(__name__)
+
+def excludeKeyInRecursiveGet(key):
+  return key == 'token'
+
+def formatResponse(data, status_code, content_type):
+  return data, status_code, {common.content_type: content_type}
+
+def formatOrderedDictContents(data):
+  def getKeyDescription(key):
+    if not excludeKeyInRecursiveGet(key):
+      if type(data[key]) is OrderedDict or isinstance(data[key], IndexedList):
```

Comment:
Done. I didn't realize you could test for multiple types that way (I'm pretty new to python). The type(...) is check was something I forgot to change over after I learned about isinstance. I wasn't trying to worry about subclasses here.